cdapython/fetch.py

```diff
@@ -1352,13 +1340,13 @@ def fetch_rows(
 
         query_for_extra_columns.l.value = ', '.join( columns_to_fetch )
 
-    elif table == 'somatic_mutation' and provenance == True:
+    elif table == 'mutation' and provenance == True:
```

Comment:
Do we still need to cover this case?

Reply:
yep, we handle `mutation` provenance data differently from the rest. see my comments for details